packages/shared/src/components/FeedLayout.tsx

```diff
@@ -80,39 +87,62 @@ const replaceDigitsWithIncrement = (str: string, increment: number): string => {
 };
 
 const sidebarRenderedWidth = 44;
-const reversedBreakpointsSidebarRendered = reversedBreakpoints.map(
-  (breakpoint) => replaceDigitsWithIncrement(breakpoint, sidebarRenderedWidth),
-);
-
 const sidebarOpenWidth = 240;
-const reversedBreakpointsSidebarOpen = reversedBreakpoints.map((breakpoint) =>
-  replaceDigitsWithIncrement(breakpoint, sidebarOpenWidth),
-);
-
-const reversedSettings = baseFeedSettings.reverse();
-
-const determineBreakPoints = ({
-  sidebarExpanded,
-  sidebarRendered,
-}: DetermineFeedSettingsProps): string[] => {
-  if (sidebarRendered) {
-    return sidebarExpanded
-      ? reversedBreakpointsSidebarOpen
-      : reversedBreakpointsSidebarRendered;
-  }
-  return reversedBreakpoints;
-};
 
 export default function FeedLayout({
   children,
 }: FeedLayoutProps): ReactElement {
   const { sidebarExpanded } = useContext(SettingsContext);
   const { sidebarRendered } = useSidebarRendered();
+  const feedPageSizes = useFeature(feature.feedPageSizes);
```

Comment:
We'll need to make sure this load appropriate in time, or we need to delay return maybe?